File: src/test/scala/cern/colt/matrix/tint/impl/SparseIntMatrix2DViewTest.scala
package cern.colt.matrix.tint.impl

//remove if not needed
import scala.collection.JavaConversions._

class SparseIntMatrix2DViewTest(arg0: String) extends SparseIntMatrix2DTest(arg0) {

  protected def createMatrices() {
    A = new SparseIntMatrix2D(NCOLUMNS, NROWS).viewDice()
    B = new SparseIntMatrix2D(NCOLUMNS, NROWS).viewDice()
    Bt = new SparseIntMatrix2D(NROWS, NCOLUMNS).viewDice()
  }
}

File: src/test/java/cernx/colt/matrix/tint/impl/SparseIntMatrix2DViewTest.java
package cernx.colt.matrix.tint.impl;

public class SparseIntMatrix2DViewTest extends SparseIntMatrix2DTest {

    public SparseIntMatrix2DViewTest(String arg0) {
        super(arg0);
    }

    protected void createMatrices() throws Exception {
        A = new SparseIntMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        B = new SparseIntMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        Bt = new SparseIntMatrix2D(NROWS, NCOLUMNS).viewTranspose();
    }
}
